src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf8Utility.Validation.cs

```diff
@@ -719,5 +732,16 @@ internal static unsafe partial class Utf8Utility
             scalarCountAdjustment = tempScalarCountAdjustment;
             return pInputBuffer;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static ulong Arm64MoveMask(Vector128<byte> value, Vector128<byte> initialMask, Vector128<byte> mostSignficantBitMask)
```

Comment:
>Arm64MoveMask [](start = 29, length = 13)

I would just rename this method to `GetNonAsciiBytes()` or something as it shouldn't be named based on API of different architecture. Another reason to name it what it does is so in future we don't wonder why `Arm64MoveMask()` in this file is different than the one in `Utf16Utility` even though both are named `Arm64MoveMask`.

Reply:
Sure, no problem. Renamed.